### PR TITLE
Crud articles and comments with validations and basic auth

### PR DESCRIPTION
**Pull request by LFTPadilla**
Implemented all the functionalities taught in RoR guide.

### CODE REVIEW

File: app/helpers/comments_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
Comment (MrJuanGaviriaK) [inline app/helpers/comments_helper.rb:1]: You should delete unused files.

File: app/models/article.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Article < ApplicationRecord
+
+  include Visible
+
+  has_many :comments, dependent: :destroy
+
+  validates :title, presence: true
+  validates :body, presence: true, length: { minimum: 10 }
+
```
Comment (MrJuanGaviriaK) [inline app/models/article.rb:9]: You should not leave a double-blank line.

File: app/models/concerns/visible.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+module Visible
+  extend ActiveSupport::Concern
+
+  VALID_STATUSES = ['public', 'private', 'archived']
+
+  included do
+    validates :status, inclusion: { in: VALID_STATUSES}
+  end
+
```
Comment (MrJuanGaviriaK) [inline app/models/concerns/visible.rb:9]: Same here.

File: app/models/concerns/visible.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  class_methods do
+    def public_count
+      where(status: 'public').count
+    end
+  end
+
```
Comment (MrJuanGaviriaK) [inline app/models/concerns/visible.rb:16]: Same here.

**Review by MrJuanGaviriaK — CHANGES_REQUESTED** [review]
Hi

I left some minor changes.

File: app/helpers/comments_helper.rb (hunk 1 above)
Comment (LFTPadilla) [reply app/helpers/comments_helper.rb:1]: I delete the others (ArticlesHelper, ApplicationHelper) too.

File: app/models/article.rb (hunk 2 above)
Comment (LFTPadilla) [reply app/models/article.rb:9]: Ok, I review other files and remove that extra blank line.

**Review by MrJuanGaviriaK — APPROVED** [review]

**Pull request merged**